### DISCUSSION TITLE
@import "mixins/banner"; @include bsBanner("");

### DISCUSSION THREAD

**Discussion by BigBenJr** [twbs/bootstrap#38495] [Q&A] [answered]
Hi.

I do not understand the purpose of these 2 first lines in node_modules/bootstrap/scss/bootstrap.scss

```
@import "mixins/banner";
@include bsBanner("");
```

Instead of doing an import of bootstrap.scss I would like to import only the components I need for my project. But as I don't know the purpose of mixins/banner I don't know if I should import it or not. The file does not seem to contains anything interesting ? But I am really novice as I just start using Bootstrap and SCSS. Could someone explain me the purpose of these 2 first lines ?

Thanks a lot !

**Comment by patkon** [comment, marked as answer]
A look into the file `_banner.scss` shows that "only" a copyright/License block is created here. Does not affect the functionality, but in my opinion should not be removed :-)
